### PR TITLE
Add script for html5 notification of completed checks on PRs

### PR DESCRIPTION
**Pull request by giorrrgio**
Add script for html5 web notification of completed checks on a PR

### CODE REVIEW

**Comment by demerzel3** [comment]
Please update README and CHANGELOG

File: github/notify-completed-checks.user.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// ==UserScript==
+// @name        Notify PR Checks ready
+// @namespace   http://adespresso.com/
+// @version     0.1
+// @grant       none
+// @include      /^https://github.com/adespresso/adespresso/pull/\d+/
+// @require      https://code.jquery.com/jquery-latest.min.js
+// @require      https://gist.github.com/raw/2625891/waitForKeyElements.js
+// @require      https://raw.githubusercontent.com/ttsvetko/HTML5-Desktop-Notifications/master/dist/Notification.min.js
```
Comment (maxcanna) [inline github/notify-completed-checks.user.js]: I suggest `https://cdnjs.cloudflare.com/ajax/libs/notify.js/2.0.3/notify.min.js`

File: github/notify-completed-checks.user.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// @updateURL    https://raw.githubusercontent.com/adespresso/monkey-tools/master/github/notify-completed-checks.user.js
+// ==/UserScript==
+var found = false;
+
+function alertReady() {
+    if (!found) {
+        window.Notification.requestPermission();
+        var text = $('h1.gh-header-title .js-issue-title').text().trim();
+        var n = new Notification( 'PR ' + text + ' is Ready, all checks passed',{icon: $("[name=icon]").val() });
```
Comment (maxcanna) [inline github/notify-completed-checks.user.js]: You've to check for permission, I suggest
```js
var text = $('h1.gh-header-title .js-issue-title').text().trim();
var notification = new Notify(text, {
        body: 'PR is ready, all checks passed',
        icon: $("[name=icon]").val(),
    });
    if (!Notify.needsPermission) {
        notification.show();
    } else if (Notify.isSupported()) {
        Notify.requestPermission(() => {notification.show();}, () => {console.log(':(');});
    }
```

**Comment by maxcanna** [comment]
I'm currently testing it and it seems to work fine

schermata 2016-11-22 alle 10 15 56

**Comment by maxcanna** [comment]
LGTM. @daBarbe @giorrrgio and @demerzel3 could you please let me have your feedbacks on this?

schermata 2016-11-24 alle 16 58 30

**Pull request closed without merging**